OctaneVSPlugin/ConnectionSettings.cs

```diff
@@ -48,14 +48,19 @@ internal class ConnectionSettings : UIElementDialogPage
 		private bool ssologin = false;
 		private ConnectionSettingsView page;
 
+		private UserPassConnectionInfo oldUserPassConnectionInfo;
+		private long oldSsid = 1001;
```

Comment:
this is not a proper way to define constants in Java
why do we use the 1001/1002 values? There are customers in SaaS which don't kave any knowledge of this info